editor/osm_auth.cpp

```diff
@@ -280,6 +273,18 @@ string OsmOAuth::FetchRequestToken(SessionID const & sid) const
   }
 }
 
+string OsmOAuth::BuildOAuth2Url() const
```

Comment:
```suggestion
std::string OsmOAuth::BuildOAuth2Url() const
```

Reply:
In `editor/osm_auth.cpp` file all methods have `string` return type. Should I change all methods to `std::string`?

Reply:
No need, let's make it later in bulk in a separate PR.